plugin.php

```diff
@@ -130,7 +130,7 @@
 add_filter( 'init', '_add_extra_api_post_type_arguments', 11 );
 add_action( 'init', '_add_extra_api_taxonomy_arguments', 11 );
 add_action( 'rest_api_init', 'rest_register_settings', 0 );
```

Comment:
@joehoyle I think settings do need to be present, because the initialisation is also when the schema is generated?